### PR TITLE
Add support for Convection's graphQL api

### PR DESCRIPTION
**Pull request by sweir27**
This PR:
- Adds support for a `.env` file and updates the README accordingly
- Updates `graphql-tools` to the latest
- Adds support for stitching up the convection API

### CODE REVIEW

**Comment by alloy** [comment]
Sweet 💃 

One question, though, where is the env file actually loaded? Was that already being done somewhere?

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
 {
   "scripts": {
-    "start": "nodemon --ext ts --exec 'ts-node' index.ts"
+    "start": "nodemon -r dotenv/config --ext ts --exec 'ts-node' index.ts"
```
Comment (sweir27) [inline package.json:3]: @alloy it's loaded here!

Comment (alloy) [reply package.json:3]: Ahh, of course!

**Comment by alloy** [comment]
Dope 👍

**Pull request merged**